enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -280,20 +272,96 @@ does the following:
 
 1. Install the Gateway API CRDs other than the gatewayclasses CRD, namely the gateways and httproutes CRDs.
 2. Create a Subscription to install OSSM.
-3. Create a ServiceMeshControlPlane to configure OSSM (what if it already exists?; see [Open Questions](#open-questions)).
-4. Create a LoadBalancer-type service (or OSSM may create it; see [Open Questions](#open-questions)).
-5. Create a DNSRecord for the service (see the following subsection).
+3. Create a ServiceMeshControlPlane to configure OSSM.
 
 The Ingress Operator effectively owns the Gateway API CRDs.  It installs
 versions of the CRDs that are compatible with the OSSM version that it installs,
 and if the CRDs are already installed, the operator may overwrite them.
 
-When OSSM is installed, it deploys the Istio control-plane and Envoy proxy.  The
-Ingress Operator configures the ServiceMeshControlPlane to enable Gateway API
-support, which means that Istio watches Gateway and HTTPRoute objects and
-configures Envoy accordingly.  Configuring a load balancer and DNS enables
-clients to connect through Envoy to backend applications as specified by the
-Gateway and HTTPRoute objects.
+When OSSM is installed, it deploys the Istio control-plane.  The Ingress
+Operator configures the ServiceMeshControlPlane to enable Gateway API support,
+which means that Istio watches Gateway and HTTPRoute objects and configures
+Envoy accordingly.  The Ingress Operator also configures the
+ServiceMeshControlPlane to enable Istio's "[Automated Deployment](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment)" feature,
+which means that Istio automatically creates an Envoy proxy deployment and a
+LoadBalancer-type service for each Gateway.  The Ingress Operator then
+configures DNS for each such service (see [New Controller to Manage DNS Records
+for Gateway Listeners](#new-controller-to-manage-dns-records-for-gateway-listeners)).  With the Ingress Operator and Service Mesh Operator
+together configuring a load balancer and DNS, this enables clients to connect
+through Envoy to backend applications as specified by the Gateway and HTTPRoute
+objects.
+
+Following is the ServiceMeshControlPlane CR that the Ingress Operator creates:
+
+```yaml
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: gwapi
+  namespace: openshift-ingress
+spec:
+  profiles:
+  - gateway-controller
```

Comment:
Just a reminder to remove profile configuration, as a result of our discussion on OSSM 2.3 vs. 2.4, if we have to go with OSSM 2.3, the gateway-controller profile wont' be available, and it's not really providing a whole lot of value.

Reply:
Here's what I have for a SMCP, just for reference:
```
apiVersion: maistra.io/v2
kind: ServiceMeshControlPlane
metadata:
  name: gwapi
  namespace: openshift-ingress
spec:
  tracing:
    type: None
  proxy:
    accessLogging:
      envoyService:
        enabled: true
      file:
        name: /dev/stdout
  policy:
    type: Istiod
  gateways:
    egress:
      enabled: false
    ingress:
      service:
        type: LoadBalancer
      enabled: true
      ingress: true
  addons:
    grafana:
      enabled: false
    kiali:
      enabled: false
    prometheus:
      enabled: false
    jaeger:
      name: jaeger
      install: {}
  security:
    manageNetworkPolicy: false
  techPreview:
    controlPlaneMode:
      ClusterScoped
  runtime:
    components:
      pilot:
        container:
          env:
            PILOT_ENABLE_GATEWAY_API: "true"
            PILOT_ENABLE_GATEWAY_API_STATUS: "true"
            PILOT_ENABLE_GATEWAY_API_DEPLOYMENT_CONTROLLER: "true"
            PILOT_GATEWAY_API_CONTROLLER_NAME: openshift.io/gateway-controller
  version: v2.3
```